### ISSUE TITLE
Results on ImageNet-LT

### ISSUE THREAD

**Issue by Vanint** [GT-RIPL/UNO-IC#1]
Hi, I try to reimplement the method on ImageNet-LT by changing the dataloader, but the results are quite poor (43.4 accuracy for training 90 epochs and 45.5 accuracy for training 200 epochs), while the baseline model is about 45.  

Do you have any results on ImageNet-LT? I wonder what's wrong for my implementation.

**Comment by PotatoTian** [comment]
Hi, 
I don't have results for ImageNet-LT right now. Our method is a post-processing method applicable to most classifiers.  It should improve on the vanilla models you trained on. You can refer to rebalancing.py for details. You will need the prior distribution of ImageNet-LT classes as input.  Could you give me more details about your implementation? If our method indeed does not work well for ImageNet-LT, it will be interesting to know the cause. Thanks!

Best
